.golangci.yml

```diff
@@ -770,7 +768,7 @@ linters-settings:
     # Enforce using attributes only (overrides no-mixed-args, incompatible with kv-only). Default: false
     attr-only: false
     # Enforce using methods that accept a context. Default: false
-    context-only: false
```

Comment:
is this change since it no longer works with the newer version of the linter and `context` is the newer field that we should be using?

Reply:
correct -- `context` config changed, there are now more options than true or
false. "" disables it to match the previous behavior

(this is also in the commit message 😃)